src/main/java/org/craftercms/studio/api/v1/service/configuration/ServicesConfig.java

```diff
@@ -16,11 +16,7 @@
 package org.craftercms.studio.api.v1.service.configuration;
 
 
-import org.craftercms.studio.api.v1.to.ContentTypeConfigTO;
-import org.craftercms.studio.api.v1.to.CopyDependencyConfigTO;
-import org.craftercms.studio.api.v1.to.DeleteDependencyConfigTO;
-import org.craftercms.studio.api.v1.to.DmFolderConfigTO;
-import org.craftercms.studio.api.v1.to.FacetTO;
+import org.craftercms.studio.api.v1.to.*;
```

Comment:
AFAIK our code convention says no * imports

Reply:
I didn't knew that, will fix it